### DISCUSSION TITLE
Victory website Victory Line - How is it implemented?

### DISCUSSION THREAD

**Discussion by JRWinter1** [FormidableLabs/victory#2578] [Q&A] [answered]
Hey all!

I'd really like to replicate the victory chart displayed on the home screen of the website:
![image](https://user-images.githubusercontent.com/65553035/227227888-55baa659-36c3-4aa9-8e33-cc8721860743.png)

I'd like a victory line of numerical values (downloads in the above example) and some vertical lines representing events in time (versions in the above example). Does anyone know the best way to implement this? Inspecting the react components on the website it looks like they use lots of VictoryLine components to acheive it. I would have thought a it was a mixture of bar chart and lines but doesn't look that way.

**Comment by adamhari** [comment, marked as answer]
https://github.com/FormidableLabs/victory/blob/main/docs/src/partials/home/hero-demo.js
